.travis.yml

```diff
@@ -33,7 +33,8 @@ env:
 jobs:
   include:
     - stage: release
-      env: SCALA_VERSION
+      env:
+        - SCALA_VERSION_2_11=2.11.11
+        - SCALA_VERSION_2_12=2.12.4
       script:
-        - docker-compose run sbt bash -c "SCALA_VERSION=2.12.4 ./build/release.sh"
-        - docker-compose run sbt bash -c "SCALA_VERSION=2.11.11 ./build/release.sh"
\ No newline at end of file
+        - docker-compose run release
```

Comment:
now it's single command, previous release failed because of double run of `release.sh`